iina/ja.lproj/Localizable.strings

```diff
@@ -46,7 +46,7 @@
 "alert.fatal_error" = "致命的なエラー: %@ \nアプリケーションを終了します。";
 "alert.filter.incorrect" = "フィルタの設定時にエラーが起きました。パラメータの形式を確認してください。";
 "alert.filter_hwdec.abort" = "破棄";
-"alert.filter_hwdec.message" = "ビデオフィルターが適用されますが、お使いのハードウェアデコーダが対応していない場合がございます。フィルタを使用しないように設定するか、二倍のメモリを使用する自動（コピー）方式に切り替えて使用することができます。\n以前に設定した操作（クロップの設定など）を適用するには、再設定する必要がある場合もあります。";
+"alert.filter_hwdec.message" = "あなたは今ビデオフィルタを適用しようとしていますが、お使いのハードウェアデコーダはビデオフィルタリングに対応していない可能性があります。ハードウェアデコーディングをオフにするか、2倍のメモリを消費する\"自動(コピー)\"をご使用ください。\\n設定後、正常に動作させるには操作をやり直す(クロップ範囲の再設定など)必要がある場合があります。";
```

Comment:
I believe there's a bug in iina-translation-utility, where it yield "\\\n". Please remove the redundant "\\", I will try to fix the bug maybe later.

Reply:
Fixed by https://github.com/iina/iina-translation-utility/commit/c5e13f365bbabe855956a034e6666dfb6d13ecec.